documentation/book/ref-loggers-KafkaBridge.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka-bridge.adoc
+
+[id='ref-loggers-KafkaBridge-{context}']
+= Kafka Bridge loggers
+
+Kafka Bridge has configurable loggers for each OpenAPI operation:
+Loggers are formatted as follows:
+
+```
+log4j.logger.http.openapi.operation.<operation-id>
```

Comment:
Just one question following the #1848 - are these loggers defined in the default file so that they can be configured inline?

Reply:
The default file has just the configuration for `healthy` and `ready` endpoints setting their log level to `WARN`.

Reply:
My understanding from #1848 is that that means that the others cannot be configured inline. Should we add them there? Or should we clarify that they can be configured only through the external configuration?